setup.py

```diff
@@ -1,27 +0,0 @@
-from Cython.Build import cythonize
```

Comment:
Please don't delete the setup.py file. It allows to install the entire sample factory as Pip package. Just remove everything related to the fast queue from it.